tools/clean.sh

```
@@ -0,0 +1,20 @@
+#!/bin/zsh
+
```

```suggestion
set -e
```

we must add this in every script so no error is ignored automatically.